2020-02-01_cw/ViewModel/MainWindowViewModel.cs

```diff
@@ -0,0 +1,73 @@
+﻿using _2020_02_01_cw.Model;
+using _2020_02_01_cw.Model.Constant;
+using _2020_02_01_cw.ViewModel.Command;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace _2020_02_01_cw.ViewModel
+{
+    public class MainWindowViewModel
+    {
+        public ObservableCollection<Product> Products { get; set; }
+        public Product SelectedProduct { get; set; }
+        public ICommand DefaultCommand { get; set; }
+        public ICommand DeleteCommand { get; set; }
+
+        public MainWindowViewModel()
+        {
+            this.Products = new ObservableCollection<Product>()
+            {
+                new Product("Asus", "PRIME H310M-E R2.0", "D:/2020-02-01_cw/2020-02-01_cw/Model/Images/asus.png", "110", "China", "Black", "0.5"),
```

Comment:
Object creation should have placed in a separate class with appropriate responsibilities.

Reply:
-1 to concerns separation.